tools/run-command.js

```diff
@@ -1,19 +1,19 @@
 #!/usr/bin/env node
 const { exec } = require('child_process');
 
+const util = require('util');
+
 function runCommand(command, dir, callback) {
     // save current pwd
     const pwd = process.cwd();
     process.chdir(dir);
```

Comment:
We might want to use [path.join](https://nodejs.org/api/path.html#path_path_join_paths) or [path.resolve](https://nodejs.org/api/path.html#path_path_resolve_paths) to make path to the dest more explicit